eng/common/TestResources/New-TestResources.ps1

```diff
@@ -52,12 +33,30 @@ param (
     [string] $Location = '',
 
     [Parameter()]
-    [ValidateSet('AzureCloud', 'AzureUSGovernment', 'AzureChinaCloud')]
```

Comment:
We will eventually need to support Azure Stack whose cloud name can be user-defined and wouldn't belong in this set.